### PR TITLE
Add cmdline interface

### PR DESCRIPTION
**Pull request by stsci-hack**
This provides a basic command line interface for use by end users or a processing pipeline.  The command defined will be:
    `runsinglehap <poller file>`
where the "<poller file>" will be a database query file with one line for each exposure to be included in the single visit products.

### CODE REVIEW

File: drizzlepac/runsinglehap.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+:License: :doc:`LICENSE`
+
+USAGE: runsinglehap [-d] inputFilename
+
+Python USAGE:
+    python
+    from drizzlepac import runsinglehap
+    runsinglehap.process(inputFilename,debug=True)
+
```
Comment (mdlpstsci) [inline drizzlepac/runsinglehap.py]: Just a typo here: "process" should be "perform".

File: drizzlepac/runsinglehap.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+The '-d' option will run this task in DEBUG mode producing additional outputs.
+
```
Comment (mdlpstsci) [inline drizzlepac/runsinglehap.py]: I would move this line up to be beneath the command line USAGE.

**Comment by mdlpstsci** [comment]
Just to small changes to docstring.

File: drizzlepac/runsinglehap.py (hunk 2 above)
Comment (stsci-hack) [reply drizzlepac/runsinglehap.py]: Fixed (along with another typo in the doc strings) in c569ae17

File: drizzlepac/runsinglehap.py (hunk 1 above)
Comment (stsci-hack) [reply drizzlepac/runsinglehap.py]: Resolved in c569ae17

**Pull request merged**